### DISCUSSION TITLE
anyone got any way to unblock extensions(added)

### DISCUSSION THREAD

**Discussion by EmpoweredPhoenix** [3kh0/ext-remover#428] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I want to add a couple of extensions to my Chromebook but the dev mode is blocked can someone help me

**Comment by Aka-but-nice** [comment, marked as answer]
There is not really a way to do this without unenrolling, CompactCow did release something to try to make this work, but it was only for people who do not have a block list.

**Reply by EmpoweredPhoenix** [reply]
ok thx

**Reply by CheezeKake77** [reply]
Block list for what? Extensions?

**Reply by Aka-but-nice** [reply]
Nooooooooooo, blocklist for chromebooks
